pkg/operator/ceph/object/s3-handlers.go

```diff
@@ -35,24 +36,32 @@ type S3Agent struct {
 	Client *s3.S3
 }
 
-func NewS3Agent(accessKey, secretKey, endpoint string, debug bool) (*S3Agent, error) {
+func NewS3Agent(accessKey, secretKey, endpoint string, debug bool, TLSEnabled bool) (*S3Agent, error) {
```

Comment:
parameters should be lower case
```suggestion
func NewS3Agent(accessKey, secretKey, endpoint string, debug bool, tlsEnabled bool) (*S3Agent, error) {
```